pantheon_advanced_page_cache.module

```diff
@@ -25,7 +25,7 @@ function pantheon_advanced_page_cache_emit_cache_tags($tags) {
     // The string might have cut of in the middle of a tag.
     // So now find the the last occurence of a space and cut to that length.
     $tags_string = substr($tags_string, 0, strrpos($tags_string, ' '));
-    watchdog('pantheon_advanced_page_cache', 'More cache tags were present than could be passed in the Surrogate-Key HTTP Header due to length constraints. To avoid a 502 error the list of surrogate keys was trimmed to a maximum length of 25,000 bytes. Removing keys beyond 25,000 means that this page will not be cleared from cache when the removed keys are cleared (usually by entity save operations). See https://www.drupal.org/project/pantheon_advanced_page_cache/issues/2911747 for more information about how you can filter out redundant or unnecessary cache metadata.', array(), WATCHDOG_WARNING);
+    watchdog('pantheon_advanced_page_cache', 'More cache tags were present than could be passed in the Surrogate-Key HTTP Header due to length constraints. To avoid a 502 error the list of surrogate keys was trimmed to a maximum length of 25,000 bytes. Removing keys beyond 25,000 means that this page will not be cleared from cache when the removed keys are cleared (usually by entity save operations). See https://www.drupal.org/project/pantheon_advanced_page_cache/issues/2973861 for more information about how you can filter out redundant or unnecessary cache metadata.', array(), WATCHDOG_WARNING);
```

Comment:
> Removing keys beyond 25,000 means that this page will not be cleared from cache when the removed keys are cleared (usually by entity save operations)

This sounds a bit odd. Maybe something like

> Since keys beyond the 25,000 maximum were removed this page will not be cleared from the cache when any of the removed keys are cleared (usually by entity save operations) as they have been stripped from the surrogate key header.